docs/source-pytorch/cli/lightning_cli.rst

```diff
@@ -2,9 +2,25 @@
 
 .. _lightning-cli:
 
-############################
-Eliminate config boilerplate
```

Comment:
I preferred the previous title. 
To me, "Eliminate boilerplate" sounds more inviting to someone to try this feature.